### PR TITLE
update interface and version for fastembed

### PR DESCRIPTION
**Pull request by generall**
- Allow iterators in fastembed
- Update to v0.1.1
- Allow parallel inference

### CODE REVIEW

File: qdrant_client/async_qdrant_fastembed.py [hunk 1]
```diff
@@ -125,15 +142,62 @@ def _scored_points_to_query_responses(
...
+        ids_accumulator: list,
+    ) -> Iterable[models.Record]:
+        if ids is None:
+            ids = iter(lambda: uuid.uuid4().hex, None)
+        if metadata is None:
+            metadata = iter(lambda: {}, None)
+        vector_name = self.get_vector_field_name()
+        for idx, meta, (doc, vector) in zip(ids, metadata, encoded_docs):
+            ids_accumulator.append(idx)
+            payload = {"document": doc, **meta}
```
Comment (joein) [inline qdrant_client/async_qdrant_fastembed.py:159]: if you get picky, here we can actually overwrite "document" if `meta` has such a field

Comment (generall) [reply qdrant_client/async_qdrant_fastembed.py:159]: yes, I also don't like this, but we can't change it anymore

Comment (generall) [reply qdrant_client/async_qdrant_fastembed.py:159]: somehow missed it in last release

File: qdrant_client/qdrant_fastembed.py [hunk 2]
```diff
@@ -137,15 +143,65 @@ def _scored_points_to_query_responses(
...
+            vector_field_name: models.VectorParams(
+                size=embeddings_size,
+                distance=distance,
+                on_disk=on_disk,
+                quantization_config=quantization_config,
+                hnsw_config=hnsw_config,
+            )
+        }
+
     def add(
```
Comment (joein) [inline qdrant_client/qdrant_fastembed.py:195]: return type is not updated in the docstring

Comment (generall) [reply qdrant_client/qdrant_fastembed.py:195]: check

File: qdrant_client/async_qdrant_fastembed.py [hunk 3]
```diff
@@ -202,12 +258,18 @@ async def add(
...
-            for (idx, payload, vector) in zip(ids, payloads, embeddings)
+        inserted_ids: list = []
+        records = self._records_iterator(
+            ids=ids, metadata=metadata, encoded_docs=encoded_docs, ids_accumulator=inserted_ids
+        )
+        self.upload_records(
+            collection_name=collection_name,
+            records=records,
+            wait=True,
+            parallel=parallel or 1,
```
[the remark this answers is not in the document]
Comment (generall) [reply qdrant_client/async_qdrant_fastembed.py:269]: check

**Pull request merged**